civihr_employee_portal/features/civihr_default_mail_content/civihr_default_mail_content.strongarm.inc

```diff
@@ -15,89 +15,63 @@ function civihr_default_mail_content_strongarm() {
   $strongarm->api_version = 1;
   $strongarm->name = 'user_mail_password_reset_body';
   $strongarm->value = '<p>
-        A request has been made to reset your CiviHR password. To reset your password, please click the button below. The link will take you to the system’s user page where you can enter a new password into the password fields. Once saved, this will be your new password to login to the site.
+        A request has been made to reset your CiviHR password. To reset your password, please click the button below. The link will take you to your CiviHR user account page where you can enter a new password.
       </p>
       <p>
-        After setting your password, you will be able to login to CiviHR  by using the following credentials:
-      </p>
-      <p>
-        Username: [user:name]
-        <br/>
-        Password: (the password you have just created)
-      </p>
-      <p>
-        Please note: The button below can only be used once to login. It expires after one day. Nothing will happen if it\'s not used - your password will not change if you do not reset it.
-      </p>
-      <p>
-        Best wishes,
-        <br/>
-        The CiviHR Team
+        Please note: the button below expires after one day and can only be used once. Nothing will happen if it\'s not used - your password will remain unchanged.
```

Comment:
Not sure about the grammar, It is usually the link that expires and not the button.

Reply:
A button that expires does sound a bit strange, but Mike said it's fine:

![image](https://user-images.githubusercontent.com/6374064/40487617-a64e2a2c-5f5c-11e8-91f5-da66363f28cc.png)